frontend/views/pages/DesignSystem.vue

```diff
@@ -777,7 +777,7 @@ export default {
       console.error('unimplemented, try using the actual Login.vue modal')
     },
     openModal (mode) {
-      sbp('okTurtles.events/emit', LOAD_MODAL, mode)
+      sbp('okTurtles.events/emit', OPEN_MODAL, mode)
```

Comment:
What do you think about adding an example of "subcontent" modals to DS? (And complete it with other scenarios (different backgrounds as I mentioned before),